packages/components/src/panel/stories/index.js

```diff
@@ -84,6 +84,26 @@ export const withIcon = () => {
 	);
 };
 
+export const CustomToggleIcons = () => {
+	const bodyTitle = text( 'Body Title', 'My Block Settings' );
+	const rowText = text( 'Row Text', 'My Panel Inputs and Labels' );
+	const iconOpen = boolean( 'Open Icon', true ) ? closeSmall : undefined;
+	const iconClosed = boolean( 'Closed Icon', true ) ? plus : undefined;
```

Comment:
We should either:

- use a `select` control to give users a few icons to choose from
- use `boolean` controls, but call them `customIconExpanded` and `customIconCollapsed`